### PR TITLE
Fix punctuation mistake

### PR DESCRIPTION
**Pull request by bogem**

### CODE REVIEW

File: README-ruRU.md [hunk 1]
```diff
@@ -3081,7 +3081,7 @@
 * <a name="consistent-string-literals"></a> Постарайтесь внедрить единообразных
   стиль кавычек для строчных литералов. В среде программистов на Руби есть два
   популярных стиля, оба из них считаются приемлемыми. Стиль **А** подразумевает
-  одинарные кавычки по умолчанию, а стиль **B** двойные кавычки.
+  одинарные кавычки по умолчанию, а стиль **B** - двойные кавычки.
```
Comment (arbox) [inline README-ruRU.md]: Please use typographically more appropriate mdash.

Comment (bogem) [inline README-ruRU.md]: I'll correct it

**Comment by bogem** [comment]
@arbox Done

**Comment by arbox** [comment]
Thank you @bogem , merged that!

**Pull request merged**